Module.yaml

```diff
@@ -0,0 +1,85 @@
+version: 1.0.0
+name: Melita IO for devices controller
+description: Module for communication with melita.io API for devices controller
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_melita_output.png
+type: process
+tags:
+  - Melita.io
+categories:
+  - id: category
+    name: Ouput
```

Comment:
For type `output` we have categories `Egress` or `Alert` for now, so please change to `category: name: Egress`. Or, we can create a new category `Controller`

Reply:
set it to Egress